graph/patterns/operations.md

```diff
@@ -33,32 +37,112 @@ You can consider related patterns such as [long running operations](./long-runni
 
 - Microsoft Graph supports the use of optional parameters. The optional parameter annotation can be used instead of creating function or action overloads when unnecessary.
 
-- API designer **MUST** use POST to call operations on resources.
+- API designer **MUST** use POST to call actions on resources.
+- API designer **MUST** use GET to call functions on resources.
+
+- The addition of a new mandatory not-nullable parameter to an existing action or function is a breaking change and is not allowed without proper versioning that is in accordance with our [deprecation guidelines](https://github.com/microsoft/api-guidelines/blob/vNext/graph/deprecation.md).
 
-- The addition of a new mandatory not-nullable parameter to an existing action or function is a breaking change and is not allowed without proper versioning.
+## Examples
 
-## Example
+### A user wants to forward email
 
 ```
- <Action Name="createUploadSession" IsBound="true" ags:EnabledForPassthrough="true" ags:OwnerService="Microsoft.Exchange">
-        <Parameter Name="bindingParameter" Type="Collection(graph.attachment)" />
-        <Parameter Name="AttachmentItem" Type="graph.attachmentItem" Nullable="false" />
-        <ReturnType Type="graph.uploadSession" />
-      </Action>
+POST https://graph.microsoft.com/v1.0/me/messages/AQMkADNkMmMxYzIwLWJkOTItNDczZC1hNmYyLWUwZjk2ZTljMDQyNQBGAAAD1dY5iRo4x0_pEqop6hOrQAcAeGCrbYV1-kiG-z9Rv6yHMgAAAgEJAAAAeGCrbYV1-kiG-z9Rv6yHMgABRxeUKgAAAA==/forward
+
+{
+    "comment": "FYI",
+    "toRecipients": [
+        {
+            "emailAddress": {
+                "address": "alex.darrow@microsoft.com",
+                "name": "Alex Darrow"
+            }
+        }
+    ]
+}
+```
+Response:
+```
+HTTP/1.1 202 Accepted
+
+    "cache-control": "private",
+    "client-request-id": "ca2d0416-a2c1-05af-df60-0921547a86e9",
+    "content-length": "0",
+    "request-id": "8b53016f-cc2b-4d9f-9818-bd6f0a5e3cd0"
+```
+
+`forward` operation is modeled as an asynchronous action bound to the Graph `event` entity type because the operation represents a complex business logic processed on the server side. 
```

Comment:
Nitpick: The text and JSON example talks about formwarding a message, however the CSDL example has action bound to evebt